test/function/samples/dynamic-import-inlining/main.js

```diff
@@ -1 +1,5 @@
-export { x as y } from './foo';
+export var x = 41;
+
+import( './foo' ).then(foo => {
```

Comment:
Similar to transparent-dynamic-inlining, maybe we should export and check the promise (see comment there)